client/src/components/SignUp.style.js

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components';
+
+export const StyleAll = styled.div`
+  background-color: #f9d52f;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column wrap;
+`;
+
+export const Title = styled.h1`
+  fontfamily: Work Sans;
+  fontstyle: Black;
+  fontsize: 38px;
+  background: white;
+  margin: 50px;
+`;
+export const StyledForm = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column wrap;
+`;
+export const InputContainer = styled.div``;
+export const StyledLabel = styled.label`
+  font-size: 18px;
+  text-align: center;
+  color: black;
+  font-weight: bold;
+`;
+export const StyledText = styled.h1`
+  font-size: 18px;
+  text-align: center;
+  color: black;
```

Comment:
use color var

Reply:
`var ` it is new for me 
I saw an example of its `var(--header-color, blue);`
I want to ask you `--header-color` we can name any name or what??